sdk/eventhub/event-hubs/test/eventHubConsumerClient.spec.ts

```diff
@@ -225,6 +225,44 @@ describe("EventHubConsumerClient", () => {
       await producerClient.close();
     });
 
+    describe("#close()", function(): void {
+      it("stops any actively running subscriptions", async function(): Promise<void> {
+        const subscriptions: Subscription[] = [];
+        const client = new EventHubConsumerClient(
+          EventHubConsumerClient.defaultConsumerGroupName,
+          service.connectionString,
+          service.path
+        );
+
+        // Spin up multiple subscriptions.
+        for (const partitionId of partitionIds) {
+          subscriptions.push(
+            client.subscribe(partitionId, {
+              async processError() {
```

Comment:
Also, our expectation would be this _never_ gets called, right? This is intended to be a graceful shutdown?

Reply:
Added a test for this!